tracer/src/Datadog.Trace/IAST/IastModule.cs

```diff
@@ -39,11 +40,12 @@ public IastModule()
             return null;
         }
 
-        return GetScope(Tracer.Instance, algorithm, integrationId, VulnerabilityTypeName.WeakHash, OperationNameWeakHash, iast);
+        return GetScope(algorithm, integrationId, VulnerabilityTypeName.WeakHash, OperationNameWeakHash, iast);
     }
 
-    private static Scope? GetScope(Tracer tracer, string evidenceValue, IntegrationId integrationId, string vulnerabilityType, string operationName, Iast iast)
+    private static Scope? GetScope(string evidenceValue, IntegrationId integrationId, string vulnerabilityType, string operationName, Iast iast, bool taintedFromEvidenceRequired = false)
```

Comment:
Yes, it will be true in the case of SqlI, but since I rolled back the PR, the parameter is not set as true for now. It will be used in the next PR with the SqlI, since we found the source of the falkyness in the sqlI test